docs/development/getting-started.rst

```diff
@@ -84,6 +84,66 @@ Alternately, you can use our ``tox`` job:
 
     $ tox -e docs
 
+Building a Mac application
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The officially supported method of building of the application depends on the
+system python, `pyobjc`_, and `py2app`_ libraries installed with Mac OS.
+`Travis-CI`_ is configured to build the mac application and run its tests.
+
+To build the application and run its tests locally use the following commands.
+
+.. code-block:: console
+
+   $ cd /dir/where/mimic/lives/
+   $ ./build-app.sh
+
+
+Once built, ``mimic.app`` can be found in the ``./dist`` directory.
+This application can be treated like any other mac application and moved into
+``~/Applications``.
+To start ``mimic``, use the open command with the path to ``mimic.app``
+, e.g. ``open ./dist/mimic.app``.
+
+When the application is running, the letter ``M`` will be visible in the
+menubar. To quit the application, simply click on the ``M`` and select
+``Quit``. You can view the application logs by opening
+``Applications/Utilities/Console.app``.
+
+To run ``mimic.app``'s tests use
+
+.. code-block:: console
+
+   $ /path/to/mimic.app/Contents/MacOS/run-tests
+
+The application can also built as a standalone application
```

Comment:
Building with the system python will produce a much smaller app (3.3M vs. 11M) so it seems like it might be worth keeping around, especially given that one of the selling points of Mimic is its small footprint :).